Cargo.toml

```diff
@@ -1,28 +1,21 @@
 [package]
-name = "cosmogony"
-version = "0.5.3"
+name = "cosmogony_builder"
+version = "0.6.0"
```

Comment:
Just to underline: it starts right off at `0.6` version. However, it sounds like a good idea to match `cosmogony_builder` version with `cosmogony`'s.